### DISCUSSION TITLE
blog/note-taking-is-new-electronic-game

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#33] [Announcements]
# blog/note-taking-is-new-electronic-game

哲学片段与学术之路

https://pathos.page/blog/note-taking-is-new-electronic-game

**Comment by imjeff** [comment]
哈哈哈哈，相信不少人会有代入感，似曾相识。
现在的GTD、日程软件、笔记软件、双链知识管理和各类博客程序，都是工具控的新型玩具。
2023年看到很多人会列自己的 APPs list，一个简单事务可能用上多个软件/APP。写个文章也多个软件之间来回切换，挺费心神的。
我也是时刻提醒自己，不要掉入这个“恶性循环”之中。

**Comment by ghost** [comment]
但这也值得庆幸，我们没有病入膏肓，折腾linux desktop,NAS,炼丹,vps（诸如此类无生产意义的折腾）。

应记住工具是为我们达成目的服务的，而不是为了折腾而折腾。博客的前端就是HTML+CSS，交互体验上适当添加JS，最早期的博客就是纯HTML，这并不妨碍技术大佬的写作；其实现在选择实在是太多了，能选数据库、选headlessCMS、编辑器、路由方式、UI、前端框架等，其实大部分折腾其实是停留在技术选型上，例如，如果我们不从php切换到JS，保持使用和技术发展的路径依赖性，php难道现在差到不能用了？我想是Wordpress无法像大部分人所想的一样实现预期，然后回到hugo等静态站点生成器SSG上，这是因为这些SSG框架背后有开源的主题模板，我认为梳理下这些折腾的路径会很有趣，SSG当然无法满足强交互，如果这个时候走错了那就翻车了，从动态站点到SSG再回到headless CMS或有后端交互的服务上，也就是无法认识到自身对强交互的需求是否为真。

综上所述，问题是从一开始就没找到自己预期场景匹配的技术实现方案，可以看到别人今天可以这么实现，把昨天自己的方案给删掉了，无限循环。

不可否定的是这和工作思维是矛盾的，大部分工作是在一个既定框架内进行操作，技术的变更需要确保生产环境的稳定，又要维护另一套开发和测试版本，工作只有当“不可抗力”因素出现时，才会推动技术实现的改变。

尽管我找不到折腾效率工具的正当理由，主观上还是对这一行为保留乐观的态度，毕竟喜欢和热爱本身就很珍贵，而非功利主义。

**Reply by Pathsis** [reply]
你提到的问题，“从一开始就没找到自己预期场景匹配的技术实现方案”，实际上可能是所有折腾这些玩意儿的人所明确意识到的，然而还是无法停止这种循环。

我想对文章补充一点的是，我的“折腾这些=玩游戏”，是一个中立的描述，因为玩游戏本身没什么好指责的。大多数人，不玩这种游戏就玩那种游戏。只是这玩意儿有点儿悖论的是，它们本身与所谓效率、创作、生产力联系在一起，其实它们就是游戏。

**Comment by rebron1900** [comment]
我也折腾过不少博客平台，后来转入Ghost博客之后持续使用了四五年了，最近也写了一篇文章记录我的心路历程。
https://1900.live/wo-wei-shi-yao-you-kai-shi-shi-yong-jacmstckjia-gou-liao/

**Comment by littleyz** [comment]
其实都是在不断尝试，不断折腾中成熟起来的。掌握精髓要点以后，很多东西，知识都能融会贯通，一通百通，之后就归于平淡了。三个阶段，看山是山，看山不是山，看山还是山。

**Comment by jkjoy** [comment]
果然人类的折腾是相通的.

**Comment by soulizer** [comment]
矛盾的根源就在于：人永远是既想要占有更多的优质资源，又不想承受拥有这些资源所需要付出的代价。

所以，不断有人在一番折腾过后发出“早知平平淡淡才是真，不该执着，耗费了钱财、虚度了光阴”的感慨，也有人在平淡的日常中突然冒出“我想搞个好东西，哪怕付出许多”的想法，甚至许多人会在这两种心境中交替循环。

这种现象并不局限于对域名、博客和笔记软件的折腾。

**Comment by robustmaster** [comment]
写的很直白，很刺耳，但也是真的。但换个角度理解，折腾域名、vps之类的，其实跟折腾钓鱼、徒步没什么区别，都是一种生活方式罢了。
文中举了几个折腾的人其实是“失败者”的例子，但应该也有很多“成功者”也是爱好折腾的。

**Comment by scenery** [comment]
哈哈，估计多数博友都会中枪。总是想找到最优解，当一个看起来更好的「文具」出现：博客平台，域名，服务器等，总忍不住去折腾把玩一下，期待能提高生产效率，不找到世界上最好的工具决不罢休。毕竟哪个小男孩不想让自己拥有最酷的网站呢？

我是觉得挺有意义和乐趣，不去尝试就无法找到更好的，一个相对舒适的创作环境会让自己更有记录的欲望。但要适可而止，不是找最好的，而是自己能够接受的。

我一直在有意避免去使用有转移成本的工具，所有笔记都用纯文本或者只包含基本语法的 Markdown 记录，所以当大多数笔记软件们都在构建自己护城河的时候，我都不用去看一眼。目前用的 Hugo 在写作体验上来说其实不算太好，虽然写了一个简单 shell 脚本帮助使用和发布，但插图片还是麻烦，一些功能也无法实现。不过大概率也不会再考虑更换其它平台了，因为我能接受相对舒适就行。

看起来我似乎已经退烧，但我们都知道，未来总会出现一个让你无法拒绝的理由。又退一步说，如果你有时间和精力去折腾这些，并且能乐在其中，新型电子游戏也是很不错的消遣。普通人游戏的目的不是打得好不好，而是自己开不开心。

**Reply by Pathsis** [reply]
确实，多数人不是打这个游戏，就是玩那个游戏。博客或各种效率工具也是一种游戏。到这里都是一个中立的描述。不过，正如别的游戏会沉迷，这种游戏也会沉迷。如果认识不到这些其实是游戏，或许就不会意识到自己其实在沉迷游戏。

另外，你的博客好歹加个giscus 嘛，有时候还是挺想回复一下的。giscus 可以让很多单纯想留下自己网址的人索然无味，也能给确实想留言的人一个途径。

**Reply by scenery** [reply]
可是我已经喜欢上了单方面输出的感觉，想吐槽的时候欢迎发邮件交流～

**Comment by fishhh666** [comment]
看到开头我还以为是「你说的朋友是不是……」那种剧情，结果点进去看还真有这样一个人！因为这经历确实很有同感，看到好玩的新玩具谁不想玩。博客软件我虽然没玩，但是 SNS 我还真的各种注册，看不过来后来才放弃。每次推给我什么软件我也感觉挺有用然后一通下载，最后基本都使用频率极低。感觉大多数人都一样吧，能直面自己已经挺好了的。

**Reply by fishhh666** [reply]
PS：博客折腾好像还挺好克服的，想象一下转移评论和文章的麻烦就好了。（不过真的有人手工干数据库导评论的，我真的见过，佩服）

**Reply by jkjoy** [reply]
刚开始可能会有兴趣和精力折腾,但是变成一件重复的工作的时候就失去了乐趣

**Comment by seeyousuperman** [comment]
目前这是我第二喜欢的博客了，每一篇都直击灵魂。

**Reply by Pathsis** [reply]
受宠若惊😁。那么问题来啦，你第一喜欢的博客是什么呢？

**Reply by seeyousuperman** [reply]
https://sanlun.bike 我心中当之无愧的第一。

**Reply by Pathsis** [reply]
我也去拜读一下

**Comment by mihouta** [comment]
最后一句总结得好呀。知道自己在玩游戏就好，别真以为自己在提高效率，哈哈。
玩游戏没错，本就无意义的人生，就是个过程。

**Reply by Pathsis** [reply]
你总结得比我好😄
